crt_portal/static/sass/custom/modal.scss

```diff
@@ -85,3 +103,26 @@ body.is-modal {
     white-space: pre-wrap;
   }
 }
+
+#external-link--modal {
+  h1 {
+    @include text--heading__medium();
+    font-weight: $theme-font-weight-bold;
+    color: color($theme-color-primary-darker);
```

Comment:
Maybe I'm misremembering, but I think you can just use the class `h3__display` on the element to pull these styles in.

Reply:
D'oh. Yes, thanks! Added -- `h2` though, as per annotations